modules/examples/src/main/java/com/ibm/cloud/platform_services/iam_policy_management/v1/IamPolicyManagementExamples.java

```diff
@@ -0,0 +1,347 @@
+import com.ibm.cloud.sdk.core.http.Response;
+import com.ibm.cloud.sdk.core.service.exception.ServiceResponseException;
+import com.ibm.cloud.sdk.core.util.CredentialUtils;
+import java.util.Map;
+import java.util.Random;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Arrays;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Below are examples on how to use IAM Policy Management service.
+ *
+ * The following environment variables are assumed to be defined when running examples below:
+ *
+ * IAM_POLICY_MANAGEMENT_URL=https://iam.cloud.ibm.com
+ * IAM_POLICY_MANAGEMENT_AUTH_TYPE=iam
+ * IAM_POLICY_MANAGEMENT_AUTH_URL=https://iam.cloud.ibm.com/identity/token
+ * IAM_POLICY_MANAGEMENT_APIKEY= <YOUR_APIKEY>
+ * IAM_POLICY_MANAGEMENT_TEST_ACCOUNT_ID= <YOUR_ACCOUNT_ID>
+ *
+ * Alternatively, above envirnmonet variables can be placed in a "credentials" file and then:
+ * export IBM_CREDENTIALS_FILE=<name of credentials file>
+ *
+ */
+
+public class IamPolicyManagementExamples {
+  private static final Logger logger = LoggerFactory.getLogger(IamPolicyManagementExamples.class);
+  protected IamPolicyManagementExamples() { }
+
+  private static Random random = new Random();
+  private static final String TEST_UNIQUE_ID = String.valueOf(random.nextInt(100000));
+  private static final String TEST_USER_PREFIX = "IBMid-SDKJava";
+  private static final String TEST_USER_ID = TEST_USER_PREFIX + TEST_UNIQUE_ID;
+  private static final String TEST_SERVICE_NAME = "iam-groups";
+  private static String testAccountId = null;
+  private static String testPolicyId = null;
+  private static String testPolicyEtag = null;
+  private static String testCustomRoleId = null;
+  private static String testCustomRoleEtag = null;
+
+  static {
+    System.setProperty("IBM_CREDENTIALS_FILE", "../../iam_policy_management.env");
+  }
+
+  @SuppressWarnings("checkstyle:methodlength")
+  public static void main(String[] args) throws Exception {
+    IamPolicyManagement service = IamPolicyManagement.newInstance();
+
+    // Load up our test-specific config properties.
+    Map<String, String> config = CredentialUtils.getServiceProperties(IamPolicyManagement.DEFAULT_SERVICE_NAME);
+    testAccountId = config.get("TEST_ACCOUNT_ID");
+
+    try {
+      // begin-create_policy
+      ResourceAttribute resourceAttributeAccount = new ResourceAttribute.Builder()
+              .name("accountId")
+              .value(testAccountId)
+              .operator("stringEquals")
+              .build();
+
+      ResourceAttribute resourceAttributeService = new ResourceAttribute.Builder()
+              .name("serviceName")
+              .value(TEST_SERVICE_NAME)
+              .operator("stringEquals")
+              .build();
+
+      ResourceAttribute resourceAttributeResource = new ResourceAttribute.Builder()
+              .name("resource")
+              .value("SDK-Java-Test")
```

Comment:
I think it would also be good to use the exact same example data across all the SDKs wherever possible.